tensorly/backend/mxnet_backend.py

```diff
@@ -150,6 +156,28 @@ def norm(tensor, order=2, axis=None):
 
     return res
 
+def int(tensor):
+    return tensor.astype('int32')
```

Comment:
I agree, it is a useful feature.. Not sure whether this shouldn't be done via dtype instead. In any case, `int` is also a default function in Python. 

If we want to keep this function, we could rename it something as `tensor_as_indices` or something? For now since it's only used to convert a list to tensor, we can simply create an numpy array of dtype int_ (see comment below)

Reply:
It seems our pytorch backend's tl.tensor() will not use the input numpy array's data type but always switch to torch.float32. Maybe the default dtype should be set to None.

Reply:
but I am not sure if can we set the default to None, which causes problem in my code. Pytorch cannot do (a-b) if a is float32 and b float64. I guess this non-robustness is the reason tl.tensor() set the default dtype to be torch.float32.